www-misc/monitorix/monitorix-3.9.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit systemd user
+
+DESCRIPTION="A lightweight system monitoring tool"
+HOMEPAGE="http://www.monitorix.org/"
+SRC_URI="https://github.com/mikaku/Monitorix/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="apcupsd hddtemp httpd lm_sensors postfix"
+S="${WORKDIR}/Monitorix-${PV}"
+
+RDEPEND="dev-perl/Config-General
+	dev-perl/DBI
+	dev-perl/HTTP-Server-Simple
+	dev-perl/IO-Socket-SSL
+	dev-perl/libwww-perl
+	dev-perl/MIME-Lite
+	dev-perl/XML-Simple
+	net-analyzer/rrdtool[graph,perl]
+	dev-perl/CGI
+	apcupsd? ( sys-power/apcupsd )
+	hddtemp? ( app-admin/hddtemp )
+	httpd? ( virtual/httpd-cgi )
+	lm_sensors? ( sys-apps/lm_sensors )
+	postfix? ( net-mail/pflogsumm dev-perl/MailTools )"
+
+pkg_setup() {
+	enewgroup ${PN}
+	enewuser ${PN} -1 -1 /var/lib/${PN} ${PN}
+}
+
+src_prepare() {
+	# Put better Gentoo defaults in the configuration file.
+	sed -e "s|\(base_dir.*\)/usr/share/${PN}|\1/usr/share/${PN}/htdocs|" \
+		-e "s|\(secure_log.*\)/var/log/secure|\1/var/log/auth.log|" \
+		-e "s|nobody|${PN}|g" -i ${PN}.conf || die
+}
```

Comment:
Must call `default` or at least `eapply_user` in this phase, otherwise it fails.